Dockerfile

```
@@ -1,17 +1,18 @@
 FROM ubuntu:latest
 
 # Setup environment variables for Docker image
-ENV FC=/usr/bin/mpif90 CC=/usr/bin/mpicc CXX=/usr/bin/mpicxx \
+ENV CC=/usr/bin/mpicc CXX=/usr/bin/mpicxx \
     PATH=/opt/openmc/bin:/opt/NJOY2016/build:$PATH \
     LD_LIBRARY_PATH=/opt/openmc/lib:$LD_LIBRARY_PATH \
     OPENMC_CROSS_SECTIONS=/root/nndc_hdf5/cross_sections.xml \
```

```suggestion
    OPENMC_CROSS_SECTIONS=$HOME/nndc_hdf5/cross_sections.xml \
```
Same we extract tar archive to `tar -C $HOME -xJ`